tests/framework/extensions/provisioning/verify.go

```diff
@@ -335,3 +342,31 @@ func GetSnapshots(client *rancher.Client, localclusterID string, clusterName str
 	return snapshots, nil
 
 }
+
+// VerifySSHTests validates the ssh tests listed in the config on each node of the cluster
+func VerifySSHTests(t *testing.T, client *rancher.Client, clusterObject *steveV1.SteveAPIObject, sshTests []string, clusterID string) {
+	sshUser := clusterObject.JSONResp["sshUser"]
+
+	steveclient, err := client.Steve.ProxyDownstream(clusterID)
+	require.NoError(t, err)
+	nodesSteveObjList, err := steveclient.SteveType("node").List(nil)
+	require.NoError(t, err)
+
+	for _, tests := range sshTests {
+		for _, rancherNode := range nodesSteveObjList.Data {
+			machineName := rancherNode.Annotations["cluster.x-k8s.io/machine"]
```

Comment:
there should be a const of this string somewhere, can you use that instead of hardcoding it here?